2-mandatory/6-journey-planner.js

```diff
@@ -64,7 +64,9 @@ function checkCodeIsThere(stringText) {
 
   Hint: Use the corresponding array method to split the array.
 */
-function getTransportModes() {}
+function getTransportModes(array) {
+  return array.slice(1,3);
+}
```

Comment:
Whoa, the use of _3_ here looks dodgy. If the input data had 1 location and 5 modes of transport, this would fail. Would this work: ```return array.slice(1, array.length)```?